reference-architectures/cloud_deploy_flow/main.tf

```diff
@@ -0,0 +1,176 @@
+# Configure the Google Cloud provider
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.0"
+    }
+  }
+  provider_meta "google" {
+    module_name = "cloud-solutions/platform-engineering-cloud-deploy-pipeline-deploy-v1"
+  }
+}
+
+variable "gcp_service_list" {
```

Comment:
Maybe this is related to my other comments about enabling services, maybe?

Also, what is the point of making this a variable?

Reply:
This is so we can reduce the number of "Copy and Pasted" code. See # Enable Services  ~line 28 (could change lines as I apply changes)

Reply:
I guess I should actually change it to a local